### ISSUE TITLE
Failed to refresh accessibility: RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by dshershov** [freelensapp/freelens#1337]
### Describe the bug

when i try to connect to the cluster - getting this error:

Failed to refresh accessibility: RangeError: Maximum call stack size exceeded

### Reproduction steps

go to cluster list
click on cluster
click connect
see error Failed to refresh accessibility: RangeError: Maximum call stack size exceeded

### Screenshots

<img width="753" height="300" alt="Image" src="https://github.com/user-attachments/assets/a261a14d-e630-43ab-b2f9-83477fd9dcfe" />

### Freelens version

1.6.1

### Operating system type and version

macos 26.0.1 m4 pro

### Kubernetes cluster version used with Freelens

1.27.5

**Comment by dshershov** [comment]
PS
I try to start any versions of freelens - but it doesn't work
when I try to start Intel version- it's ok, no error, but yea, very slow working AMD64 version on ARM64 cpu
